epicweb-dev/epic-stack
# (docs) update adr template

# Summary

This PR encompasses two main changes. First, an ADR (Architecture Decision Record) has been created for the ADR itself. Second, the build steps have been automated to generate an index file.

## Test Plan

To validate the changes, please follow these steps:

1. Switch to the `main` branch: `git checkout main`
2. Fetch the latest changes: `git fetch`
3. Merge the `epic-stack:docs/update-adr-template` branch: `git merge epic-stack:docs/update-adr-template`
4. Switch to the `feat/proposal` branch: `git checkout feat/proposal`
5. Copy the `adr-template.md` file into the decisions directory using the following filename format: `NNNN-title-with-dashes.md`, where `NNNN` represents the next sequential number. For example: `cp docs/adr-template.md docs/decisions/0001-title-with-dashes.md`
6. Edit the `NNNN-title-with-dashes.md` file as needed.
7. Update the `index.md` file with: `yarn adr:update`

Once you have completed these steps, verify that the `index.md` file in the `docs/decisions` directory has been successfully updated with the newly created ADR.

## Platform update
- [x] Node Dependencies - bumping versions or adding a new dependency
